test/bats/pp/bridge-e2e.bats

```diff
@@ -64,6 +64,12 @@ setup() {
     assert_success
     local bridge_tx_hash=$output
 
+    echo "------- bridge_getBridges API testcase"
+    local aggkit_node_url=$(kurtosis port print $enclave cdk-node-001 rpc)
+    run get_bridges $aggkit_node_url $l1_rpc_network_id
```

Comment:
Can we return bridges as a return value from the `get_bridges` function and assert that some values are expected: `origin_network`, `origin_address`, `destination_network`, `destination_address`, `amount`, etc.
WDYT?